config/crd/bases/ran.openshift.io_clustergroupupgrades.yaml

```diff
@@ -39,6 +39,17 @@ spec:
           spec:
             description: ClusterGroupUpgradeSpec defines the desired state of ClusterGroupUpgrade
             properties:
+              blockingCRs:
+                items:
+                  description: BlockingCR defines the Upgrade CRs that block the current
```

Comment:
nit. maybe reword. 
BlockingCR defines the running Upgrade CRs that block the current CR from running

Reply:
I believe the blockingCRs must go under status not spec otherwise the ArgoCD/ACM App that sync the ClusterGroupUpgrade.yaml from the git repo will detect there are different and it may re-override it.

Reply:
@serngawy , the blocking CRs should be under spec as we are specifying which UOCRs block the current CR from running, it's not a status that we are updating. If we want to use the blocking feature, then they should also be present in the ClusterGroupUpgrade.yaml from the git repo. They are optional though.

Reply:
okay, if it will be defined in the ClusterGroupUpgrade before adding it to the hub that will work.